src/main/java/stsjorbsmod/cards/DimensionDoor.java

```diff
@@ -0,0 +1,50 @@
+package stsjorbsmod.cards;
+
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.potions.SmokeBomb;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.characters.Wanderer;
+
+import static stsjorbsmod.JorbsMod.makeCardPath;
+
+// 0: Escape from a non-boss combat and receive no rewards. If you escape, you may ignore paths when choosing the next room to travel to. Destroy this card.
+public class DimensionDoor extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(DimensionDoor.class.getSimpleName());
+    public static final String IMG = makeCardPath("Manipulation_Rares/dimension_door.png");
+
+    private static final CardRarity RARITY = CardRarity.RARE;
+    private static final CardTarget TARGET = CardTarget.SELF;
+    private static final CardType TYPE = CardType.SKILL;
+    public static final CardColor COLOR = Wanderer.Enums.WANDERER_GRAY_COLOR;
+
+    private static final int COST = 0;
+
+    private static boolean activatedMapTravel = false;
+
+    public static boolean isMapTravelActive() {
+        return activatedMapTravel;
+    }
+
+    public static void deactivateMapTravel() {
+        activatedMapTravel = false;
+    }
+
+    public DimensionDoor() {
+        super(ID, IMG, COST, TYPE, COLOR, RARITY, TARGET);
+        this.upgraded = true;
```

Comment:
It looks a little off for it to start out upgraded, I think, though I understand that it's awkward to come up with a good upgrade effect. Maybe upgrade to "you can choose any floor from the next 2 levels, rather than the next 1 level"?

Reply:
This turns out to be a fun effect. So many available locations! The default interaction with the boss room works well. It can't be reached in this manner, so a person won't accidentally or intentionally skip the last campfire.